core/src/test/java/apoc/export/csv/ImportCsvTest.java

```diff
@@ -67,6 +82,19 @@
                     new AbstractMap.SimpleEntry<>("id", "id:ID|name:STRING\n" +
                             "1|John\n" +
                             "2|Jane\n"),
+                    new AbstractMap.SimpleEntry<>("csvPoint", 
+                            ":ID,location:point{crs:WGS-84}\n" +
+                            "1,\"{latitude:55.6121514, longitude:12.9950357}\"\n" +
+                            "2,\"{y:51.507222, x:-0.1275}\"\n" +
+                            "3,\"{latitude:37.554167, longitude:-122.313056, height: 100, crs:'WGS-84-3D'}\"\n"),
+                    new AbstractMap.SimpleEntry<>("nodesMultiTypes",
+                            ":ID(MultiType-ID)|date1:datetime{timezone:Europe/Stockholm}|date2:datetime|foo:string|joined:date|active:boolean|points:int\n" +
```

Comment:
scusa non ho capito cos'è questo `:ID(MultiType-ID)`

Reply:
È l'equivalente di questo qua https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin-import/#import-tool-id-spaces, ovvero per definire degli id univoci nel caso di più file.
Non c'entra ovviamente con la issue, è solo per far vedere che funziona anche in questo caso